client/src/index.jsx

```diff
@@ -12,20 +12,32 @@ class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = { repos: [] }
+    this.get()
```

Comment:
AJAX requests should be done in the `componentDidMount` method because if `setState` is called before the component is mounted, the component will not update. Keeping `get` in the constructor would introduce a race condition.

[More info in this thread](https://stackoverflow.com/questions/38913138/can-i-put-ajax-in-react-component-constructor)